tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala

```diff
@@ -385,4 +385,109 @@ class Scala3DocumentHighlightSuite extends BaseDocumentHighlightSuite {
        |""".stripMargin
   )
 
+  check(
+    "i5977",
```

Comment:
Is this already fixed in the compiler? Why don't we need to ignore it?

Reply:
I think it was fixed by https://github.com/lampepfl/dotty/pull/18557

Reply:
Ach ok, thanks!